packages/dashboard-frontend/tests/services/remote-cached-data-provider.test.js

```diff
@@ -0,0 +1,78 @@
+import { describe, expect, it } from "@jest/globals";
+import { waitFor } from "@testing-library/react";
+import { fetchJson } from "../../src/fetch/fetch-json";
+import { getItem, setItem } from "../../src/services/cache";
+import { RemoteCachedDataProvider } from "../../src/services/remote-cached-data-provider";
+
+jest.mock( "../../src/services/cache", () => ( {
+	getItem: jest.fn(),
+	setItem: jest.fn(),
+} ) );
+jest.mock( "../../src/fetch/fetch-json" );
+
+describe( "RemoteCachedDataProvider", () => {
+	beforeAll( () => {
+		fetchJson.mockImplementation( () => {
+			return Promise.resolve( "result" );
+		} );
+	} );
+
+	beforeEach( () => {
+		fetchJson.mockClear();
```

Comment:
Actually looked this up just now, and it seems already done by Jest itself:
https://jestjs.io/docs/mock-function-api#mockfnmockclear

Reply:
Removed [here](https://github.com/Yoast/wordpress-seo/pull/22216/commits/0ae3d8c105a3edbf69e239be193d988feac12679).